src/components/views/messages/MessageEvent.tsx

```diff
@@ -125,19 +125,17 @@ export default class MessageEvent extends React.Component<IProps> implements IMe
                 BodyType = UnknownBody;
             }
 
-            // TODO: this can be done in eventTypes when Polls stabilise
+            // TODO: this can be done in eventTypes when Polls stabilises
```

Comment:
this comment should be updated to specify that it isn't the feature stabilising (which is happening in this PR), but the MSC/spec presumably?

Reply:
Fixed in e11106d1e8